test/e2e/utils/objects/objects.go

```diff
@@ -66,3 +73,49 @@ func TestMCP() *machineconfigv1.MachineConfigPool {
 		},
 	}
 }
+
+func TestKC(matchLabels map[string]string) *machineconfigv1.KubeletConfig {
+	return &machineconfigv1.KubeletConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "KubeletConfig",
+			APIVersion: machineconfigv1.GroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   "kc-test",
+			Labels: map[string]string{"test": "test"},
+		},
+		Spec: machineconfigv1.KubeletConfigSpec{
+			KubeletConfig: &runtime.RawExtension{
+				Raw: getEncodedKCConfig(),
+			},
+			MachineConfigPoolSelector: &metav1.LabelSelector{
+				MatchLabels: matchLabels,
+			},
+		},
+	}
+}
+
+func getEncodedKCConfig() []byte {
```

Comment:
next time please consider creating a function which just returns a fully initialized `kubeletconfigv1beta1.KubeletConfiguration` object and encode it in place in `TestKC`. This will make the code more composable